### DISCUSSION TITLE
Majority voting

### DISCUSSION THREAD

**Discussion by medha-chippa** [online-ml/river#615] [Q&A] [answered]
Is there an equivalent of the voting classifier in river or some function in river that helps with performing majority voting once predictions from a group of classifiers are obtained?

Thank you

**Comment by MaxHalford** [comment, marked as answer]
No there isn't a `VotingClassifier` exactly but it shouldn't be too hard to implement if you want to have a go at it.

**Reply by smastelini** [reply]
What about feeding the classifiers' outputs to `dummy.PrioClassifier`? Does it do the trick?

**Reply by MaxHalford** [reply]
Yes that's the idea! I'll implement this right now.

**Reply by MaxHalford** [reply]
Implementation [here](https://github.com/online-ml/river/pull/667).

**Reply by medha-chippa** [reply]
Thank you!
